wiki/Tournaments/o!BAM/1/en.md

```diff
@@ -0,0 +1,576 @@
+
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration phase | 2021-08-10/2021-08-29 |
+| Qualifiers | 2021-09-04/2021-09-05
+| Live drawings | 2021-09-06 (21:00 UTC+8) |
+| Round of 32 | 2021-09-11/2021-09-12 |
+| Round of 16 | 2021-09-18/2021-09-19 |
+| Quarterfinals | 2021-09-25/2021-09-26 |
+| Semifinals | 2021-10-2/2021-10-3 |
+| Finals week 1 | 2021-10-9/2021-10-10 |
+| Finals week 2 | 2021-10-16/2021-10-17 |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 2 months supporter per player,Animated Profile Banners, Exclusive Wrist-rest custom-designed by Wristlabs.net  |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 1 months supporter per player,Animated Profile Banners |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 1 months supporter per player,Animated Profile Banners |
+
+## Organisation
+
+The osu! Battle Archive Malaysia is run by various community members.
+
+| Position | Member(s) |
+| :-- | :-- |
+| Organizer | ![][flag_MY] [LouisZpro](https://osu.ppy.sh/users/6231292), ![][flag_MY] [Tzero](https://osu.ppy.sh/users/6088976) |
+| Administrator | ![][flag_MY] [Laphii](https://osu.ppy.sh/users/9364594), ![][flag_SG] [Ayucchi](https://osu.ppy.sh/users/7823498) |
+| Mappool selector | ![][flag_MY] [LouisZpro](https://osu.ppy.sh/users/6231292), ![][flag_MY] [Tzero](https://osu.ppy.sh/users/6088976), ![][flag_SG] [Ayucchi](https://osu.ppy.sh/users/7823498) |
+| Streamer | ![][flag_MY] [Laphii](https://osu.ppy.sh/users/9364594), ![][flag_MY] [Hand Sanitiser](https://osu.ppy.sh/users/5091293), ![][flag_MY] [Koyaiba](https://osu.ppy.sh/users/7062729) | 
+| Commentator | ![][flag_MY] [MyAngelFable](https://osu.ppy.sh/users/11234664), ![][flag_MY] [Malaidan](https://osu.ppy.sh/users/14279913), ![][flag_MY] [fancyyy](https://osu.ppy.sh/users/12616268), ![][flag_MY] [sealw4ll](https://osu.ppy.sh/users/12289818), ![][flag_MY] [Banner](https://osu.ppy.sh/users/14290988), ![][flag_MY] [MinaMyoui](https://osu.ppy.sh/users/6996926), ![][flag_MY] [IHateAutoNicks](https://osu.ppy.sh/users/10758397), ![][flag_MY] [Chiyuu](https://osu.ppy.sh/users/8226107), ![][flag_MY] [Its not Marvin](https://osu.ppy.sh/users/13600612) |
+| Referee | ![][flag_MY] [Xyrelyx](https://osu.ppy.sh/users/3765632), ![][flag_MY] [Laphii](https://osu.ppy.sh/users/9364594),![][flag_VN] [rock-on](https://osu.ppy.sh/users/9676089), ![][flag_MY] [Vexine](https://osu.ppy.sh/users/4042706), ![][flag_FR] [Taevas](https://osu.ppy.sh/users/7276846), ![][flag_FR] [DaNoobSlayaar](https://osu.ppy.sh/users/8688777), ![][flag_FR] [Wen4ng3L](https://osu.ppy.sh/users/9288833), ![][flag_MY] [Hand Sanitiser](https://osu.ppy.sh/users/5091293), ![][flag_MY] [MinaMyoui](https://osu.ppy.sh/users/6996926) |
```

Comment:
as someone who handles the wiki page of [the currently ongoing oseat5](http://osu.ppy.sh/wiki/id/Tournaments/oSEAT/5) i'm quite sure that DaNoobSlayaar isn't from France lol

Reply:
I seen quite a lot of error because I think I dint hit the save button when i was commiting